src/Akka.Hosting.TestKit/Akka.Hosting.TestKit.csproj

```diff
@@ -9,6 +9,7 @@
 
     <ItemGroup>
         <PackageReference Include="Akka.TestKit.Xunit2" Version="$(AkkaVersion)" />
+        <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="7.0.0" />
```

Comment:
I got an error inside Akka.Cluster.Hosting.Tests claiming that this assembly (Microsoft.Bcl.AsyncInterfaces) could not be found, so I added it to the TestKit project. If this was an err I can try adding it to the tests and not the shared library. I added it to the library because I suspected that other users might run into this problem.

Amazingly enough, I've used the TestKit package in my own apps (which also target .NET 6.0) and never ran into this problem. I assumed that the problem in this solution was that Akka.Hosting.TestKit targets .NET Standard 2.0.

Reply:
Microsoft.Bcl.AsyncInterfaces have a severe backward compatibility with netstandard2.0 target, I had to write my own `.FirstOrDefault()` in `Discovery.Azure` to avoid that problem.